### ISSUE TITLE
Reset untitled counter when creating a new form

### ISSUE THREAD

**Issue by lognaturel** [opendatakit/build#129]
Create a new form, add a few controls which will be numbered untitled1, untitled2, etc. Open a new form and the numbering will continue where it left off rather than restarting. 

This isn't a big deal but it's a little bit surprising. Maybe it'd make for a good quick win for a new contributor?

**Comment by clint-tseng** [comment]
Yeah, I know of this behaviour. It's not quick or I'd have fixed it.
